rasa/core/channels/socketio.py

```diff
@@ -175,8 +201,17 @@ async def health(_: Request) -> HTTPResponse:
             return response.json({"status": "ok"})
 
         @sio.on("connect", namespace=self.namespace)
-        async def connect(sid: Text, _) -> None:
-            logger.debug(f"User {sid} connected to socketIO endpoint.")
+        async def connect(sid: Text, environ: Dict) -> None:
```

Comment:
according the docs there is also an `auth` parameter (https://python-socketio.readthedocs.io/en/latest/server.html#defining-event-handlers) which seems to be the better fit: